roles/provision-keycloak-apb/tasks/configure-keycloak.yml

```diff
@@ -0,0 +1,127 @@
+---
+- name: "Generate keycloak auth token"
+  uri:
+    url: "{{ KEYCLOAK_HOST }}/auth/realms/master/protocol/openid-connect/token"
+    method: POST
+    body: "client_id=admin-cli&username={{ ADMIN_NAME }}&password={{ ADMIN_PASSWORD }}&grant_type=password"
+    validate_certs: no
+  register: keycloak_auth_response
+  until: keycloak_auth_response.status == 200
+  retries: 600
+  delay: 2
+
+- name: Generate namespace specific realm
+  template:
+    src: keycloak_realm.json.j2
+    dest: /tmp/keycloak_realm.json
+
+- name: "Create {{ KEYCLOAK_REALM }} realm"
```

Comment:
what happens here if the realm already exists? It is a use case that the user may wish to reuse an existing realm

Reply:
Done!